### PR TITLE
[fix] ScopeBuilder operator for hasOne

### PR DESCRIPTION
**Pull request by ibelar**
Fix issue where hasOne condition is used. This fix will make sure that the proper operator is selected.

```
$model = new Stat($app->db, ['caption' => 'Demo Stat']);
$model->addCondition('client_country_iso', '=', 'Anguilla');
```

Will show in Ui like in picture below where operator dropdown is not set:

Screen Shot 2020-10-20 at 11 34 32 AM

@georgehristov - please have a look and see if this fix is enough or if something else is needed.

### CODE REVIEW

File: src/Form/Control/ScopeBuilder.php [hunk 1]
```diff
@@ -218,7 +218,7 @@ class ScopeBuilder extends Control
         'enum' => [
             'type' => 'select',
             'operators' => [
-                self::OPERATOR_EQUALS,
+                self::OPERATOR_TEXT_EQUALS,
```
Comment (mvorisek) [inline src/Form/Control/ScopeBuilder.php]: what is the difference and why the condition below is not updated as well?

Comment (ibelar) [reply src/Form/Control/ScopeBuilder.php]: Was not sure myself, so this is why I asked @georgehristov to take a look.

Comment (ibelar) [reply src/Form/Control/ScopeBuilder.php]: It does fix the issue but we may have to look further.

**Comment by georgehristov** [comment]
I am looking into this. It needs a better fix...

**Comment by mvorisek** [comment]
If this fixes... I would like to see Behat test for this, as ScopeBuilder is completely untested currently.

**Comment by georgehristov** [comment]
Yes, a test is necessary ... will think and work on it

**Review by mvorisek — APPROVED** [review]
fix seems clean, but I do not know all internals of the ScopeBuilder class

**Pull request merged**